.gitattributes

```diff
@@ -1,6 +1,11 @@
 /.github/                       export-ignore
 /.phive/                        export-ignore
 /.editorconfig                  export-ignore
+/src/ExampleBench.php           export-ignore
+/src/ExampleTest.php            export-ignore
+/src/Helper.php                 export-ignore
+/src/ValueCanNotBeBlankTest.php export-ignore
+/src/ValueCanNotBeEmptyTest.php export-ignore
```

Comment:
When you work on a package and declare test code in the same directory as production code, adding exclusions for test code in `.gitattributes` is a lot of work.